### ISSUE TITLE
Windows DD agent Version 6.1.1 - Issues

### ISSUE THREAD

**Issue by SIVA451** [DataDog/datadog-agent#1577]
While testing the agent V6 on windows, I have seen that agent not opening in the browser.

As a part of the testing, I have installed the 5.6v later downloaded and executed the EXE installer to update the agent to the newer version. After this, we have downloaded and installed the agent 6.1.1 from https://github.com/DataDog/datadog-agent/releases.

Kindly, help us to fix this. Thanks in advance.

**Comment by arbll** [comment]
Hi @SIVA451,

Could you describe more precisely what you mean by the "agent is not opening in the browser" ? Are you having troubles opening the web server based GUI ?

**Comment by SIVA451** [comment]
Yes,  I am using the PowerShell to automate the process to upgrade the v5 to v6 accordingly.

**Comment by arbll** [comment]
As described in https://app.datadoghq.com/account/settings#agent/windows, you should try to upgrade from 5.6 to a version that is >= 5.12.0 but < 6.0.0 before making the jump to v6. Please try this and tell me if it fix your issues

**Comment by SIVA451** [comment]
I am using EXE installer if it is <=5.12 to upgrade it to the latest version. I have raised the bug on another page with DD team without answering this they closed the case.

https://github.com/DataDog/docker-dd-agent/issues/303

**Comment by arbll** [comment]
Could you give me more details about the issue itself ? What appends when you run `datadog-agent launch-gui` ? Does the agent show up at https://app.datadoghq.com/infrastructure ?

**Comment by arbll** [comment]
Could you describe in more details what you mean by "is not launching" ? To access the GUI you need to run `datadog-agent launch-gui` in an elevated command prompt.

**Comment by SIVA451** [comment]
It is not launching in the browser and running on the backend. After running the command.
